### PR TITLE
Package inyourface

### PR DESCRIPTION
**Pull request by yacomink**
This is a PR to make inyourface work as a pip module so we can separate out the bits to run as a service.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import setuptools
+from inyourface.version import Version
+
+
+setuptools.setup(name='inyourface',
+                 version=Version('0.0.1').number,
+                 description='In Your Face',
+                 long_description=open('README.md').read().strip(),
+                 author='Andy Yaco-Mink',
+                 author_email='andy@yacomink.com',
```
Comment (goldsamantha) [inline setup.py]: Welp now i'm in a hole and, taco, this is horrifying: https://www.buzzfeed.com/yacomink/crushable-buscemi?utm_term=.nkweJjjL5#.wt4QBggj9

**Review by goldsamantha — COMMENTED** [review]
This is sweet! I've never made a pip module before but i'm exccitteeddd. woeoooo. I think this is a really good idea and something that it seems like we've both been reaching for in the way that we've been including the library in other projects! I'm stoked

File: tests/helpers/InMemoryCacheProvider.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class CacheProviderForTests(object):
+
```
Comment (goldsamantha) [inline tests/helpers/InMemoryCacheProvider.py:2]: heya, might be too broad a question, but i'm curious what this necessary for?

**Review by goldsamantha — COMMENTED** [review]
This PR is *chef kiss*. Are there anythings you're thinking about doing next or that I can help with?

**Comment by yacomink** [comment]
I think the main thing it still needs is the file path stuff for overlays,
if you're interested in doing that across all the effects. I also _suspect_
the images directory is going to have to move into the inyourface directory
to get packaged up ... But I haven't figured out how static assets work for
pip yet.

The cache provider stuff is so that the test setup doesn't need to
setup/teardown a sqlite db. Doing it in memory makes it easier to ensure
you start with a clean slate for each test run. Plus having a test-specific
class let me drop in easy hooks to check when it's been called and with
what. Mocking out all of DefaultCacheProvider's interface would have
accomplished the same thing, but this seemed easier.

**Comment by goldsamantha** [comment]
cool! I'm happy to do the file path stuff! 

Also thanks for the explanation--you da real mvp

**Pull request merged**